### DISCUSSION TITLE
演算法幫助貝多芬寫完了他逝世前未完成的《第十號交響曲》

### DISCUSSION THREAD

**Discussion by BlahFish1006** [1101-GS4719-C/Python-Community-GS4719-C#1020] [科技新知]
貝多芬1827年逝世後，留下了未竟之作第十號交響曲，一專家團隊將貝多芬的草稿、筆記及其生活時代的樂譜，輸入到人工智能系統中，由演算法「以貝多芬的想法」續寫完成了這個作品，更在貝多芬音樂節上由波恩貝多芬管弦樂團首次演出。

文中有位音樂學教授萊文說：「你可以說，電腦是用演算法做到的。沒錯，但是人類依據的也是經驗和訓練。兩者的差距其實不是太大。」大家是怎麼想的呢？我個人認為能利用技術為一個遺憾下一個句號也是件挺棒的事 (本音樂小白反正是聽不出什麼差別XD)

原文連結：https://www.storm.mg/amparticle/3985523
大家也可以親自來聽聽看「原版貝多芬」和「人工智能作曲」之間是否存在差異：https://youtu.be/Rvj3Oblscqw (宜配合期末考試食用)

**Comment by Tuoresa** [comment]
感覺能夠用這樣的方式完成未完成的遺憾是一件好令人感動的事！
能把遺憾用最好的方式圓滿的感覺！
不過作曲本身實在是非常倚賴個人想法的事情，根據作曲當下心情、環境等都可能改變譜出的音符
而我們能從音樂中認識到的作曲家也僅僅是一個面向而已，
還是好希望能聽見作曲家本人親手寫下的故事。

**Comment by ghost** [comment]
我覺得不論是演算法或是專家，都無法代表貝多芬的想法，因為人的一生成就了一個人，不是了解簡單資料或是習慣就能瞭解的，所以只能說演算法提供了新的一種十號交響曲的可能性，但協助寫完只是無稽之談

**Comment by Valentinaa0214** [comment]
透過演算法技術分析貝多芬之前寫的樂譜，我們可以了解貝多芬是如何寫出樂曲。但使用演算法寫出來的樂譜可能與貝多芬本人寫出來的樂譜完全不一樣，用演算法來推敲個人創意的這個想法對我而言不太現實。
不知道貝多芬知道了會不會氣到從棺材跳出來(X

**Comment by s96015welson** [comment]
我認為作品是由創作者所賦予靈魂的，即使AI的作品比貝多芬的作品更好，其依然無法替代貝多芬的作品，也無法代表貝多芬。而這也是我認為許多未完成的作品，就算被後人甚至是AI完成了，大概都還是很難超越原作者。

**Comment by HsuChen88** [comment]
每個人寫字都會有一定的習慣，走路、說話、寫曲也一樣，說不定AI寫出來的曲子與貝多芬真的相差不遠。

**Comment by Jzd314** [comment]
AI完成的作品跟貝多芬應該還是有一些差距，才子總是在一些地方會突破常規，才會讓人感到驚艷

**Comment by harrylee1971** [comment]
人類跟AI的不同在於創造力，AI只可以吸收出現過的東西，但是我們人類可以發明出新型的事情!!!
